randovania/games/super_metroid/logic_database/Brinstar.txt

```diff
@@ -978,28 +1002,28 @@ Extra - asset_id: 100
   * Layers: default
   * Normal Door to Noob Bridge/Door to Red Tower
   > Door to Red Brinstar Fireflea Room
-      Morph Ball and Power Bombs
+      Can Lay Power Bombs
   > Door to Hellway
       Any of the following:
           Space Jump
+          # Freeze enemies to use as platforms
           Hi-Jump Boots and Ice Beam
           All of the following:
-              Wall Jumps (Intermediate)
-              Super Missile ≥ 4 or Enabled Red Tower Tweaks
-          # https://www.youtube.com/watch?v=7bHxm5TttRI
+              Wall Jumps (Advanced)
+              Any of the following:
+                  Super Missile ≥ 4
+                  # Defeat enemies before jumping up
+                  Mid-Air Morph (Intermediate) and Can Lay Power Bombs
```

Comment:
i dont quite understand this. what enemies, the rippers? are they vulnerable to PBs?

Reply:
The enemies are rippers and they are vulnerable to pbs. It is easiest to move them out of the way before attempting the wall jump.